frontend/src/types.ts

```diff
@@ -12,6 +12,16 @@ export interface UserType {
     toolbar_mode: string
 }
 
+export interface PersonalAPIKeyType {
+    id: string
+    label: string
+    value?: string
+    created_at: string
+    last_used_at: string
```

Comment:
Is mixing snake case and camel/pascal case common throughout the project? That seems unfortunate.

I realize the backend is in Python which wants snake case. This can typically be solved by adding a thin middleware abstraction which converts pascal/camel case into snake case. There are other solutions as well.

Not a huge deal, but seems like a refactor worth doing eventually.